cattle/plugins/docker/compute.py

```diff
@@ -613,9 +609,9 @@ def _setup_networking(self, instance, host, create_config, start_config):
         setup_links(instance, create_config, start_config)
         setup_ipsec(instance, host, create_config, start_config)
 
-    def _is_privileged(self, instance):
+    def _is_true(self, instance, key):
         try:
-            return instance.data.fields['privileged']
+            return instance.data.fields[key]
```

Comment:
Nit: For the impl to align more closely with the function name, it would make more sense to do 
```
return instance.data.fields[key] is True
```
But, I won't fight leaving it as-is